lib/cocoapods/generator/copy_dsyms_script.rb

```diff
@@ -35,10 +40,14 @@ def generate
 #{Pod::Generator::ScriptPhaseConstants::STRIP_INVALID_ARCHITECTURES_METHOD}
 #{Pod::Generator::ScriptPhaseConstants::RSYNC_PROTECT_TMP_FILES}
 #{Pod::Generator::ScriptPhaseConstants::INSTALL_DSYM_METHOD}
+#{Pod::Generator::ScriptPhaseConstants::INSTALL_BCSYMBOLMAP_METHOD}
         SH
-        dsym_paths.each do |dsym_path|
+        dsym_paths&.each do |dsym_path|
```

Comment:
we canot use `&` due to Ruby < 2.3 support. I took over your PR to finish it off.